README.md

```diff
@@ -1,9 +1,9 @@
 # APIS EventManager
 [![Build Status](https://travis-ci.com/furthemore/APIS.svg?branch=production)](https://travis-ci.com/furthemore/APIS) [![Coverage Status](https://coveralls.io/repos/github/furthemore/APIS/badge.svg?branch=production)](https://coveralls.io/github/furthemore/APIS?branch=production)
 
-Data Model: http://www.gliffy.com/go/publish/10890219
+Data Model: http://www.gliffy.com/go/publish/10890219 (broken link?)
 
-Registrtion Workflows: http://www.gliffy.com/go/publish/10892367
+Registration Workflows: http://www.gliffy.com/go/publish/10892367 (broken link?)
```

Comment:
I unfortunately do not have the source graph for this image anymore, we can go ahead and remove this.